adafruit_ble_berrymed_pulse_oximeter/adafruit_ble_transparent_uart.py

```diff
@@ -76,7 +82,7 @@ def readinto(self, buf, nbytes=None):
         """
         return self._rx.readinto(buf, nbytes)
 
-    def readline(self):
+    def readline(self) -> Union[int, None]:
```

Comment:
Here's another place to substitute with `Optional`

Reply:
It also looks like the return type should be `Optional[bytes]` instead of `Optional[int]`, since the return is the actual line that was read.